hocon.gemspec

```diff
@@ -1,13 +1,14 @@
 Gem::Specification.new do |s|
   s.name        = 'hocon'
-  s.version     = '1.1.2'
+  s.version     = '1.1.2-SNAPSHOT'
```

Comment:
Just curious on this one - I don't think gems formally support / understand the SNAPSHOT concept, though I have no real objection to using it anyway.  I do think we'd technically be on 1.1.3-SNAPSHOT or 1.2.0-SNAPSHOT though, since 1.1.2 is already released, right?